gnomad/sample_qc/ancestry.py

```diff
@@ -148,7 +148,7 @@ def assign_population_pcs(
         can be used to expand this column into multiple `PC` columns.
 
     :param pop_pca_scores: Input Hail Table or Pandas Dataframe
-    :param pc_cols: Columns storing the PCs to use
+    :param pc_cols: List of which PCS to use/columns storing the PCs to use (i.e. [1,2,4,5])
```

Comment:
```suggestion
    :param pc_cols: List of which PCs to use/columns storing the PCs to use (i.e. [1,2,4,5])
```

Reply:
Should also give an example if using a pandas dataframe? Also, we should note if this indexing is 0-based or 1-based (I think from the code it looks one based)